internal/component/database_observability/README.md

```diff
@@ -339,3 +339,23 @@ prometheus.scrape "database_observability_mysql_example_db_2" {
   forward_to = [prometheus.remote_write.metrics_service.receiver]
 }
 ```
+
+## Setting up the Postgres database
+
+1. Your Postgres DB should be version 16.
```

Comment:
is it greater than or equal to version 16?

Reply:
We've not tested on 14 or 15 so this is fine for now. But agree we'll want to revise this in future